scripts/image_builder.sh

```diff
@@ -34,6 +34,10 @@ die()
 	exit 1
 }
 
+info() {
```

Comment:
this is just a comment about style, the other function use the braces below of function name, could be better to use `function` keyword

Reply:
+1